packages/kbn-panel-loader/index.tsx

```diff
@@ -9,14 +9,14 @@
 import React from 'react';
 import { EuiLoadingChart, EuiPanel } from '@elastic/eui';
 
-export const EmbeddableLoadingIndicator = ({ showShadow }: { showShadow?: boolean }) => {
+export const PanelLoader = (props: { dataTestSubj?: string }) => {
```

Comment:
I think we may still need the `showShadow` prop here. Without it, the Dashboard panel load looks slightly different before & after the panel is loaded.

BEFORE:
![Before](https://github.com/elastic/kibana/assets/14276393/fda22b2d-e43e-4401-8509-9f09f548534d)

AFTER:
![After](https://github.com/elastic/kibana/assets/14276393/dc874f34-3bec-4883-bb47-c92724fa225d)

Reply:
I couldn't see where the `showShadow` prop was used. That's why I felt comfortable removing it.

However, I'm thinking that the current behavior (on main) is relying on `showShadow` being `undefined` which `EuiPanel` interpreted as true. Now that I have changed the default to `false`, the style is broken.

I will look into passing this as a prop only on dashboard 👍 .

Reply:
Ah that makes sense, it would be way more explicit with the prop passes as `true` there. Thanks!